src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -69,6 +70,31 @@ def __init__(
 
         self.num_channels = config.model.num_channels
 
+        # If self.layer_kernels entry is missing from the config, use torch.nn by default
```

Comment:
can you factor this out to a separate member function? the body of the init is rather long now

Reply:
done, _load_layer_kernels()